<launch>
  <arg name="robot_namespace" default="base1"/>
  <arg name="dataset_path"    default="/data/LoopComputeBenchmark/PointCloudData" />
  <arg name="test_name"       default="test7" />
  <arg name="output_dir"      default="/data/LoopComputeBenchmark/Outputs" />

  <group ns="$(arg robot_namespace)">

    <node pkg="loop_closure"
          name="eval_loop_computation_test"
          type="eval_loop_computation_test"
          output="screen"> 
      <param name="dataset_path"   value="$(arg dataset_path)" />
      <param name="output_dir"     value="$(arg output_dir)" />
      <param name="test_name"      value="$(arg test_name)" />
      <rosparam file="$(find lamp)/config/lamp_settings.yaml" subst_value="true"/>
      <rosparam file="$(find loop_closure)/config/laser_parameters.yaml" subst_value="true"/>     
      <rosparam file="$(find lamp)/config/precision_parameters.yaml" subst_value="true"/> 
      <!-- Point cloud filter -->
      <rosparam file="$(find point_cloud_filter)/config/lamp_parameters.yaml"/>
    </node>

  </group>

</launch>
